examples/static-spark-encrypted/README.md

```diff
@@ -0,0 +1,61 @@
+This example depends on an existing VPC and subnet with configurations that meets [EMR cluster requirements](https://aws.amazon.com/blogs/big-data/launching-and-running-an-amazon-emr-cluster-inside-a-vpc/).
+
+# Cloudwatch log collection example for an EMR Static Spark cluster
+
+Pre-requisites:
+- In case in-transit encryption is enabled there is a need to ensure that the bucket hosting the PEM file allows
+S3 actions comming from external resources like for instance, EMR cluster.
+
+What does the example do?
+
+The following example will deploy the necessary resources on every instance that belongs to an EMR Cluster in order to:
+- Download the Cloudwatch agent.
+- Install the Cloudwatch agent.
+- Create and mount the configuration file.
+- Enable the Cloudwatch agent.
+
+<!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
+## Requirements
+
+No requirements.
+
+## Providers
+
+| Name | Version |
+|------|---------|
+| aws | n/a |
```

Comment:
We need to specify versions for this module to avoid breaking changes

Reply:
It was restricted in the root.


`terraform {
  required_version = ">= 0.13"
  required_providers {
    aws = ">= 3.36.0, < 4.0.0"
  }
}
`